src/codegen.cpp

```diff
@@ -331,11 +331,12 @@ struct jl_tbaacache_t {
     // LLVM should have enough info for alias analysis of non-gcframe stack slot
     // this is mainly a place holder for `jl_cgval_t::tbaa`
     MDNode *tbaa_stack;      // stack slot
-    MDNode *tbaa_unionselbyte;   // a selector byte in isbits Union struct fields
+    MDNode *tbaa_immut_unionselbyte;   // a selector byte in isbits Union struct fields
```

Comment:
These comments are organized (and indentation is formatted) to represent the tree of the TBAA hierarchy, so any updates should preserve that